.github/workflows/switch_env.yaml

```diff
@@ -0,0 +1,106 @@
+name: Switch EKS FINOS Legend Prod and Staging environments
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Switch Legend Prod and Staging environments
+    runs-on: ubuntu-latest
+    steps:
+      - name: Installing Dependencies
+        run: |
+          sudo snap install juju --channel=3.0/stable --classic
+
+      # We need the kubeconfig and the controllers.yaml file into our environment
+      # in order to login and switch the environments.
+      - name: Adding Credentials
+        shell: bash
+        run: |
+          mkdir -p ~/.kube
+          mkdir -p ~/.local/share/juju
+          echo "${KUBECONFIG_B64}" | base64 -d > ~/.kube/config
+          echo "${CONTROLLERS_B64}" | base64 -d > ~/.local/share/juju/controllers.yaml
+
+          echo "${CONTROLLER_PASSWORD}" | juju login -c finos-legend-v3 -u admin
+
+        env:
+          KUBECONFIG_B64: "${{ secrets.KUBECONFIG_B64 }}"
+          CONTROLLERS_B64: "${{ secrets.CONTROLLERS_B64 }}"
+          CONTROLLER_PASSWORD: "${{ secrets.CONTROLLER_PASSWORD }}"
+
+      - name: Switch environments
+        run: |
+          # Controller name is "finos-legend-v3", model name is "finos-legend"
+          juju switch finos-legend-v3:finos-legend
+
+          # Running juju status will show us the current revisions of the charms.
+          echo "finos-legend model status:"
+          juju status --relations
+          echo "finos-legend-twin status:"
```

Comment:
For consistency, this should also be `finos-legend-twin model status:`

Reply:
Done